components/fxa-client/src/http_client.rs

```diff
@@ -307,6 +315,27 @@ impl FxAClient for Client {
         Ok(())
     }
 
+    fn listAttachedOAuthClients(
+        &self,
+        config: &Config,
+        session_token: &str,
+    ) -> Result<AttachedClientsResponse> {
+        let body = json!({
+            "client_id": clientId
+            "device_id": deviceId
+            "sessionTokenid": sessionTokenId
+            "refresh_token": refreshTokenId
```

Comment:
These aren't the correct arguments: https://github.com/mozilla/fxa/blob/master/packages/fxa-auth-server/docs/api.md#get-accountattached_clients

`clientId`, not `client_id`, etc.